common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -247,6 +253,7 @@ def get_html(self):
             'element_id': self.location.html_id(),
             'num_inputs': self.num_inputs,
             'submitted': self.submitted,
+            'js_includes': js_includes,
```

Comment:
I don't know if there are negative consequences of this, or what the state of Studio/XBlock interaction is these days.